vbr/vbrCore/functions/PiezometerWH2006.m

```diff
@@ -0,0 +1,62 @@
+function d = PiezometerWH2006(sig)
```

Comment:
for function input/output variable names, I like to have the units in the variable names (though I'm not always good at remembering to do so):

```suggestion
function d_um = PiezometerWH2006(sig_MPa)
```

(I also added code suggestions below to change the variable names where needed)